### PR TITLE
Add topics for first couple of exercises

### PR DESCRIPTION
**Pull request by devonestes**
In the issue it recommended that we add topics for each exercise one by one,
but for these first ones I think we can chunk them up a little more, so I'm
doing like 5 or 6 at a time for each PR. I may even do more per PR if folks
feel that it's a lot to review so many PRs - just let me know!

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -74,36 +74,49 @@
       "slug": "hello-world",
       "difficulty": 1,
       "topics": [
+        "IO",
```
Comment (parkerl) [inline config.json]: I don't think there is any IO in this one?

File: config.json [hunk 2]
```diff
@@ -74,36 +74,49 @@
...
+        "strings"
       ]
     },
     {
       "slug": "nucleotide-count",
       "difficulty": 1,
       "topics": [
+        "maps",
+        "string processing"
```
Comment (parkerl) [inline config.json]: Also `enumerables`?

File: config.json (hunk 1 above)
Comment (devonestes) [inline config.json]: Oh, yeah, in the Elixir version we don't actually output that to the console. Good catch!

File: config.json (hunk 2 above)
Comment (devonestes) [inline config.json]: seems like a good addition to me!

**Comment by parkerl** [comment]
looks good! 🎉

**Pull request merged**